Formula/handbrake.rb

```diff
@@ -0,0 +1,28 @@
+class Handbrake < Formula
+  desc "Open-source video transcoder available for Linux, Mac, and Windows."
+  homepage "https://handbrake.fr/"
+  url "https://github.com/HandBrake/HandBrake.git", :revision => "72e77efa1c309be6a5e525a9484ec131707af848"
+  version "1.0.0-alpha1"
+  head "https://github.com/HandBrake/HandBrake.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "yasm" => :build
+
+  def install
+    system "./configure", "--debug=none",
+                          "--disable-xcode",
+                          "--disable-gtk"
+    cd "build" do
+      system "make"
+      bin.install "HandBrakeCLI"
```

Comment:
I don't think so. cc: @ilovezfs  Last time I checked, didn't seem like it. Referring to: https://handbrake.fr/docs/en/latest/developer/build-mac.html